src/daemon/daemon.cpp

```diff
@@ -3071,6 +3071,9 @@ void mp::Daemon::create_vm(const CreateRequest* request,
                         reply.set_vm_instance_name(name);
                         config->update_prompt->populate_if_time_to_show(reply.mutable_update_info());
 
+                        // Set the zone name from the vm_instance_specs
+                        reply.set_zone_name(vm_instance_specs[name].zone);
+
```

Comment:
Hmm, while this works, it might be better to capture the zone in the capture list of this lambda, just like the name.